EVPurchaseProgram/src/main/java/ro/siit/evprogram/DiscountedPrice.java

```diff
@@ -0,0 +1,22 @@
+package ro.siit.evprogram;
+
+import java.util.ArrayList;
+
+public class DiscountedPrice {
+
+    GreenBonusProgram gbp = new GreenBonusProgram();
+
+    /**
+     * Method for calculating the price after using the discount
+     * @param prices
+     * @return
+     */
+
+    public int getDiscountedPrice(ArrayList<PriceList> prices) {
```

Comment:
Why do you need a separate class for this? Why isn't it part of the CarDealership class?